src/tree/tree_manager.rs

```diff
@@ -137,7 +137,7 @@ impl TreeVersion for CanonicalTree {
                     .await?;
 
                     for update in identity_updates {
-                        tracing::info!(?chain_id, new_root = ?update.0.hash, "Root updated");
+                        tracing::info!(?chain_id, new_root = ?update.0.hash, "Canonical root updated");
```

Comment:
Can we keep this as `Root updated`? We're filtering by this log message in DataDog